testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -583,6 +613,15 @@ public void testCommentView()
         assertAccessAllowed("COMMENT ON VIEW " + viewName + " IS 'new comment'");
     }
 
+    @Test
+    public void testCommentOnRedirectedTable()
+    {
+        @Language("SQL") String query = "SELECT * FROM system.metadata.table_comments WHERE catalog_name = 'mock' AND schema_name = 'default' AND table_name LIKE 'redirected%'";
```

Comment:
`@Language("SQL")` is redundant to get SQL syntax highliting in intelliJ (IDE infers this based on the usage)